guides/common/assembly_managing-users-and-roles.adoc

```diff
@@ -34,7 +34,7 @@ include::modules/proc_creating-a-complete-permission-table.adoc[leveloffset=+2]
 
 include::modules/proc_removing-a-role.adoc[leveloffset=+2]
 
-include::modules/ref_predefined-roles.adoc[leveloffset=2]
+include::modules/ref_predefined-roles.adoc[leveloffset=+2]
```

Comment:
This one is now hidden, lost )5.4.7. Predefined Roles Available in Foreman - what's on nightly branch looks better